wd_demo.au3

```diff
@@ -181,7 +181,7 @@ Func RunDemo($idDebugging, $idBrowsers, $idUpdate, $idHeadless)
 	$sSession = _WD_CreateSession($sCapabilities)
 	If @error And _RunDemo_errorHandling(@error, @extended, $iWebDriver_PID, $sSession) Then Return
 
-	Local $iError
+	Local $iError = $_WD_ERROR_Success
```

Comment:
Please take a look on [Demo stopped!](https://github.com/Danp2/WebDriver/pull/176/commits/db2dea6341e5d1ce766e0c96fd00c8c7a9e0f221)

Reply:
It would be nice if you answered my question instead of referring me to another commit. 🙂 

Also, your `switch` in _RunDemo_ErrorHander appears to be written incorrectly.

Reply:
They must be initialized because there can be a case user which use wd_demo.au3 do not select any demo.
Then accurate value should be passed to `_RunDemo_ErrorHander` .
Then I notice that `_RunDemo_ErrorHander` doesnt handle such case so I initialize `Local $iError = -1`

Reply:
> Also, your `switch` in _RunDemo_ErrorHander appears to be written incorrectly.

Thanks.
Fixed with: [_RunDemo_ErrorHander() switch fix](https://github.com/Danp2/WebDriver/pull/176/commits/9c49513f955f749fd6f515615d85a7f0af8b1d11)

Reply:
See https://github.com/Danp2/WebDriver/pull/176/commits/bdfe16e77e2bdb6e1182d2ea77ae540441179d00 for an alternate solution. We can revert mine if you don't like it. 🥲

Reply:
I didn't think about it, and this is a better solution.
Thanks.

Reply:
Take a look: [disable button - moved](https://github.com/Danp2/WebDriver/pull/176/commits/c77d3e3a909571aa60653dae8e76bc2b11bbc4a1)

Reply:
@mlipok That's fine as long as it doesn't reintroduce the issue where the button flickers when you move the mouse.

Reply:
it was reverted 
[disable button - moved - revert](https://github.com/Danp2/au3WebDriver/pull/176/commits/27abc1cbb4337b7a77c4f9cd2215ea1e0f7a09f6)

so could we treat this part as "Resolved conversation" ?